tests/unit/EditShareStructure.spec.ts

```diff
@@ -74,28 +74,28 @@ function createComponent (
 }
 
 function createShareStructure (
-  id = null,
-  priority = null,
-  type,
-  name,
-  hasMaximumShares = true,
-  maxNumberOfShares = null,
-  hasParValue = true,
-  parValue = null,
-  currency = null,
-  hasRightsOrRestrictions = false
```

Comment:
These default assignments were not needed because all calls to this method supplied the necessary values.

FYI, having default assignments is mostly useful for values that are left omitted, for example if we wanted to do this:
```
const x = createShareStructure() // no formal parameters!
```